ckanext/recombinant/controller.py

```diff
@@ -469,6 +469,24 @@ def _process_upload_file(lc, dataset, upload_file, geno, dry_run):
             for f in chromo['fields']
             if ('choices' in f or 'choices_file' in f)}
 
+        # validate example row if exists
+        if example_row_values:
+            for key in chromo['examples']['record']:
+                val = example_row_values[key]
+                if chromo['examples']['record'][key] != val:
```

Comment:
I think we can be more lenient here. If we update one field in an example (changing a name to "John Smith" is a recent case) I don't think we should force everyone to download a new copy of the template. How about checking just the primary key fields or checking if they've removed the 'e.g.' or 'ex.' text in the first column?